src/content/wiki/buildings/miner.mdoc

```diff
@@ -30,6 +52,7 @@ While mining, sometimes the Miner will get lucky and get an ore block instead of
 | 4              | Bedrock       |
 | 5              | "             |
 {% /version %}
+{% /version %}
```

Comment:
Try to avoid nested version blocks if possible, this can very easily mess something up if the ranges are different